From e355b4264eb0149f35fd717566a759b3e28f38fa Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Sun, 13 Mar 2016 12:56:37 +0700
Subject: [PATCH 6/7] libbinder: fix unresolved symbol WriteString16

until bionic doesn't properly load shim libs, use this workaround
---
 libs/binder/Parcel.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libs/binder/Parcel.cpp b/libs/binder/Parcel.cpp
index 886b6b6..0e92074 100644
--- a/libs/binder/Parcel.cpp
+++ b/libs/binder/Parcel.cpp
@@ -2661,3 +2661,5 @@ void Parcel::Blob::clear() {
 }
 
 }; // namespace android
+
+__strong_alias(_ZN7android6Parcel13writeString16EPKtj,_ZN7android6Parcel13writeString16EPKDsj);
-- 
2.5.0

